rero_ils/modules/documents/commons/identifiers.py

```diff
@@ -0,0 +1,281 @@
+# -*- coding: utf-8 -*-
```

Comment:
Identifiers can be used by other type of resources such as contributions. Maybe, you can put it at the modules level?

Reply:
Moved to `rero_ils.modules.commons.py`